### PR TITLE
Website update for Apache Drill 1.19.0

### PR DESCRIPTION
**Pull request by laurentgo**
# Website update for Apache Drill 1.19.0

### CODE REVIEW

File: _docs/getting-started/010-drill-introduction.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ applications, while still providing the familiarity and ecosystem of ANSI SQL,
 the industry-standard query language. Drill provides plug-and-play integration
 with existing Apache Hive and Apache HBase deployments.   
 
+## What's New in Apache Drill 1.19
```
Comment (luocooong) [inline _docs/getting-started/010-drill-introduction.md:13]: A lot of features are missing and I will submit comments later.

**Comment by luocooong** [comment]
| Title | JIRA ID |
| --- | --- |
| Add XML Format Plugin | DRILL-7823 |
| Add Storage Plugin for Splunk | DRILL-7751 |
| Convert format plugins to EVF | DRILL-7525, DRILL-7532, DRILL-7533, DRILL-7534, DRILL-7536 |
| Merge row set-based JSON reader | DRILL-6953 |
| Use streaming for REST JSON queries | DRILL-7733 |
| Add Limit Pushdown to Storage Plugin | DRILL-7763, DRILL-7800, DRILL-7814 |
| Build Drill with Netty version 4.1.50.Final | DRILL-7790 |
| Update to 30-jre Guava version | DRILL-7904 |
| Bump Mongo driver from 3.12 to 4.2 | DRILL-7903 |
| Bump Kafka driver from 2.3 to the 2.8 | DRILL-7917 |
| Bump HBase driver from 2.2 to 2.4 | DRILL-7918 |
| Bump Kudu driver from 1.3 to 1.14 | DRILL-7918 |

File: _docs/getting-started/010-drill-introduction.md (hunk 1 above)
Comment (laurentgo) [reply _docs/getting-started/010-drill-introduction.md:13]: Does all version changes can be actually considered as new features?

Comment (laurentgo) [reply _docs/getting-started/010-drill-introduction.md:13]: Added all the new features mentioned below, left aside the version changes

Comment (luocooong) [reply _docs/getting-started/010-drill-introduction.md:13]: Everything is okay. Just, The Mongo (4.4) and Kafka (2.8) storage plugin have had a major upragde than before.

**Comment by luocooong** [comment]
@laurentgo This looks good. Could you please confirm that the [2b27a1e](https://github.com/apache/drill/pull/2257/commits/2b27a1e60f8870dacee963bd4c62febbadc21dec) revision only affect the changes for the `gh-pages/apidocs` directory?

**Comment by laurentgo** [comment]
> @laurentgo This looks good. Could you please confirm that the [2b27a1e](https://github.com/apache/drill/pull/2257/commits/2b27a1e60f8870dacee963bd4c62febbadc21dec) revision only affect the changes for the `gh-pages/apidocs` directory?

Yes, correct

**Review by luocooong — APPROVED** [review]
LGTM. +1

**Pull request merged**